spec/interactors/update_user_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe UpdateUser do
+  subject(:context) { described_class.call(permitted_params: permitted_params, current_user: current_user) }
+
+  let(:permitted_params) {}
+  let(:current_user) { create(:user) }
+
+  describe ".call" do
+    before do
+      current_user.billing = Billing.create
```

Comment:
Maybe it would be better to use trait :with_addresses in your Users factory? https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#traits